xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -393,12 +393,12 @@ bool CMusicInfoScanner::IsScanning()
   return m_bRunning;
 }
 
-void CMusicInfoScanner::Stop()
+void CMusicInfoScanner::Stop(bool wait /*= false*/)
```

Comment:
Please add a space between `/*` and `=` and after `false` for better readability.